app/connectors/CodeListConnector.scala

```diff
@@ -59,7 +59,20 @@ class FileBasedCodeListConnector @Inject()(appConfig: AppConfig) extends CodeLis
 
     locals.map { locale =>
       val procedureCodes = codeList.map(factory(_, locale))
-      (locale -> procedureCodes)
+      (locale.getLanguage -> procedureCodes)
+    }.toMap
+  }
+
+  private def readCodesMapFromFile[T <: CommonCode](srcFile: String, factory: (CodeItem, Locale) => T)(implicit locals: Seq[Locale]) = {
```

Comment:
IMHO `implicit locals: Seq[Locale]` is not useful at all. Why don't you directly use `supportedLanguages`?

Reply:
Yeah, I suppose as they are private methods we don't need to pass them as a param. Changed